guacamole-docker/bin/start.sh

```diff
@@ -303,27 +303,29 @@ END
     set_property          "ldap-hostname"           "$LDAP_HOSTNAME"
     set_optional_property "ldap-port"               "$LDAP_PORT"
     set_optional_property "ldap-encryption-method"  "$LDAP_ENCRYPTION_METHOD"
+    set_optional_property "ldap-max-search-results" "$LDAP_MAX_SEARCH_RESULTS"
+    set_optional_property "ldap-search-bind-dn"     "$LDAP_SEARCH_BIND_DN"
     set_property          "ldap-user-base-dn"       "$LDAP_USER_BASE_DN"
     set_optional_property "ldap-username-attribute" "$LDAP_USERNAME_ATTRIBUTE"
-    set_optional_property "ldap-member-attribute"   "$LDAP_MEMBER_ATTRIBUTE"
-    set_optional_property "ldap-group-base-dn"      "$LDAP_GROUP_BASE_DN"
+    set_optional_property "ldap-user-search-filter" "$LDAP_USER_SEARCH_FILTER"
     set_optional_property "ldap-config-base-dn"     "$LDAP_CONFIG_BASE_DN"
-
-    set_optional_property     \
-        "ldap-search-bind-dn" \
-        "$LDAP_SEARCH_BIND_DN"
```

Comment:
I like the reformatting overall, but I'm concerned about these items that reorganize properties away from each other - for example, the `ldap-search-bind-dn` property ends up several lines away from `ldap-search-bind-password`.  While this isn't something that users will every see or care about, from a debugging and code maintenance point of view I think it could organized so that it makes a little more sense.

Reply:
Hi Nick, I actually preferred that when making the change but opted to keep it the same as the previous formatting where the shorter lines were grouped and then the longer lines grouped after.

Just to be clear, would you prefer to have them interspersed like this?

    set_optional_property "ldap-search-bind-dn"     "$LDAP_SEARCH_BIND_DN"

    set_optional_property           \
        "ldap-search-bind-password" \
        "$LDAP_SEARCH_BIND_PASSWORD"

    set_property          "ldap-user-base-dn"       "$LDAP_USER_BASE_DN"
    set_optional_property "ldap-username-attribute" "$LDAP_USERNAME_ATTRIBUTE"


Let me know and I'll amend the PR.

Reply:
IMHO, yes, it makes more sense to group the options together that logically go together, rather than grouping them by line length.  If others want to vote me down on that, that's fine, we can discuss it, and I don't feel uber-passionate about it, but it does make more sense to me.

Reply:
I've made the changes and squashed my commits to remove the merge (I think!). Commit 26d3d51 should have the changes you requested.